addons/x_lib/functions/camera/fnc_setCameraAngle.sqf

```diff
@@ -25,48 +25,18 @@ Author:
 ARJay
 ---------------------------------------------------------------------------- */
 
-private ["_position", "_angle", "_y"];
-
-_position = _this select 0;
-_angle = _this select 1;
-
-switch(_angle) do
-{
-    case "DEFAULT":
-        {
-
-        };
-    case "LOW":
-        {
-            _y = _position select 2;
-            _position set [2,_y+1];
-        };
-    case "EYE":
-        {
-            _y = _position select 2;
-            _position set [2,_y+1.3];
-        };
-    case "HIGH":
-        {
-            _y = _position select 2;
-            _position set [2,_y+2];
-        };
-    case "BIRDS_EYE":
-        {
-            _y = _position select 2;
-            _position set [2,_y+20];
-        };
-    case "UAV":
-        {
-            _y = _position select 2;
-            _position set [2,_y+100];
-        };
-    case "SATELITE":
-        {
-            _y = _position select 2;
-            _position set [2,_y+500];
-        };
-
+params ["_position", "_angle"];
+
+private _y = _position select 2;
+
+switch(_angle) do {
+    case "DEFAULT": {};
+    case "LOW": { _position set [2,_y+1]; };
+    case "EYE": { _position set [2,_y+1.3]; };
+    case "HIGH": { _position set [2,_y+2]; };
+    case "BIRDS_EYE": { _position set [2,_y+20]; };
+    case "UAV": { _position set [2,_y+100]; };
+    case "SATELITE": { _position set [2,_y+500]; };
```

Comment:
I don't like the formatting. Either line them up vertically or but the inner block on their own line.